spec/models/room_spec.rb

```diff
@@ -54,12 +54,41 @@
     end
   end
 
-  describe 'after_create' do
```

Comment:
Why we're changing the structure?

Reply:
Ah good catch, it's because we removed the `after_create callback` and then we put it back.